packages/web-components/package.json

```diff
@@ -17,7 +17,7 @@
   "dependencies": {
     "@agoric/assert": "^0.6.0",
     "@agoric/cache": "^0.3.2",
-    "@agoric/casting": "^0.4.2",
+    "@agoric/casting": "0.4.3-dev-0001c49.0",
```

Comment:
it would help to have some process for cleaning up `dev` deps whenever a new agoric-sdk release is published. maybe part of the SDK release runbook.